cirq-core/cirq/contrib/acquaintance/executor.py

```diff
@@ -64,6 +67,9 @@ def __call__(self, *args, **kwargs):
         return StrategyExecutor(self)(*args, **kwargs)
```

Comment:
I made this change. Now executor_test.py tests the new function instead of the old one but when I run it, I am getting the error message:
TypeError: Operation without a known matrix or decomposition: cirq.contrib.acquaintance.AcquaintanceOpportunityGate(num_qubits=5).on(cirq.LineQubit(0), cirq.LineQubit(1), cirq.LineQubit(2), cirq.LineQubit(3), cirq.LineQubit(4)).

I try to print the given circuit before and after the mapping and I get something like
0: ───█───
      │
1: ───█───
      │
2: ───█───
      │
3: ───█───
      │
4: ───█───

After mapping
0: ───█───0↦0───
      │   │
1: ───█───1↦1───
      │   │
2: ───█───2↦2───
      │   │
3: ───█───3↦3───
      │   │
4: ───█───4↦4───
(using num_qubits=5 and acquaintance_size=5)

From the error message, it is failing because it is unable to represent the circuit into a unitary. Is this because of the arrows ↦ ??  From what I understand the arrows ↦ signify a mapping from a qubit to another? But I am not sure either.